ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -505,10 +505,10 @@ private void handlePrintedTopic(String printTopic)
         try {
           topicPrintFuture.get();
         } catch (CancellationException exception) {
+          topicResponse.getResponse().close();
```

Comment:
Wouldn't this change mean the response is only closed on a cancellation?

Reply:
As we declare `topicStreamScanner` in try-with-resources, I think it will also be closed on other cases.